iceoryx_hoofs/include/iceoryx_hoofs/cxx/helplets.hpp

```diff
@@ -270,8 +271,7 @@ enum class RelativePathComponents
 /// @param[in] relativePathComponents are relative path components are allowed for this path entry
 /// @return true if it is valid, otherwise false
 template <uint64_t StringCapacity>
-bool isValidPathEntry(const string<StringCapacity>& name,
-                      const RelativePathComponents& relativePathComponents) noexcept;
+bool isValidPathEntry(const string<StringCapacity>& name, const RelativePathComponents) noexcept;
```

Comment:
Removing the parameter name shouldn't be necessary. Besides, the doxygen comment refers to it.